soc/arm/nxp_lpc/lpc55xxx/Kconfig.soc

```diff
@@ -29,6 +30,7 @@ config SOC_LPC55S16
 	select CLOCK_CONTROL
 	select HAS_MCUX_IAP if !TRUSTED_EXECUTION_NONSECURE
 	select HAS_MCUX_MCAN
+	select HAS_MCUX_RNG
```

Comment:
We should focus this PR only on the LPC55S36. If we want to add the select of RNG to other platforms lets do that in a standalone PR that also connects the entropy driver.

Reply:
Hi @dleach02 , LPC55S36 don't support RNG, so I removed LPC55XXX common config HAS_MCUX_RNG in file "soc/arm/nxp_lpc/lpc55xxx/Kconfig.series", and add the select of RNG separately for other platforms. Is there any other better way to remove RNG support for LPC55S36? Thanks a lot.

Reply:
okay. That makes sense.